C++/src/ADIS16448_IMU.cpp

```diff
@@ -127,7 +127,7 @@ ADIS16448_IMU::ADIS16448_IMU(Axis yaw_axis, AHRSAlgorithm algorithm)
   m_acquire_task = std::thread(&ADIS16448_IMU::Acquire, this);
   m_calculate_task = std::thread(&ADIS16448_IMU::Calculate, this);
 
-  Calibrate();
+  if (ReadRegister(kRegXGYRO_OFF)) Calibrate();
```

Comment:
This logic is the opposite of the Java logic.  Don't we want to run Calibrate if the register contents are zero?